ext/ox/builder.c

```diff
@@ -104,7 +104,7 @@ append_indent(Builder b) {
 }
 
 static void
-append_string(Builder b, const char *str, size_t size, const char *table) {
+append_string(Builder b, const char *str, size_t size, const char *table, const bool strip_invalid_chars) {
```

Comment:
No need for a boolean to be const as it is a pass by value.

Reply:
I thought it prevented it from being assigned accidentally in the function.